File: LB/PROgram0055.c
/*

Q54) Write a PROgram which accpet number and count how much occurance of 4 in that Given number

    input :- 4025
    output :- 1

*/

#include<stdio.h>

int Count(int iNo1)
{
    int iDigit = 0;
    int iCnt = 0;

    for(; iNo1 !=0; )
    {
        iDigit = iNo1 % 10;
       
        if((iDigit >= 0) && (iDigit <= 6))
        {
            iCnt++;
        }

        iNo1 = iNo1 / 10;
    }

    return iCnt;
}

int main()
{
    int iValue1 = 0;
    int iRet = 0;

    printf("Enter the Number : ");
    scanf("%d",&iValue1);

    iRet = Count(iValue1);

    printf("Occurences of Digit which are less tha 6 in Given Number : %d",iRet);

    return 0;
}

File: PM/PROgram0055.py
'''

Q15) Write a program that checks whether the given number is palindrome or not

'''

def Palindrome(No):
    Temp = No
    Rev = 0

    while(No != 0):
        Digit = No % 10
        Rev = (Rev * 10) + Digit
        No = No // 10

    if(Temp == Rev):
        return True
    else:
        return False

def main():
    Value = int(input("Enter the Number : "))

    Ret = Palindrome(Value)

    if(Ret == True):
        print("Palindrome Number")
    else:
        print("Not a Palindrome Number")

if __name__ == "__main__":
    main()